### ISSUE TITLE
maybe target `/bin/sh`?

### ISSUE THREAD

**Issue by jbenet** [tdenniston/bish#4]
I think this is a great idea. I wonder, perhaps it may make sense to target `sh` instead of `bash`? afaik, `sh` is more portable + found in more systems. though i'm not sure.

**Comment by hachre** [comment]
+1

**Comment by ryanmjacobs** [comment]
Oooh, yes this is a good idea. Oftentimes, I'm restricted because of `sh`. If bish could compile to `sh`, it would be very useful.

**Comment by jbenet** [comment]
Yeah, i exclusively use `sh` for portability. It's painful. `bish` might really alleviate that!

**Comment by tdenniston** [comment]
This is a good idea. I'll keep bash as the default target (due to features I plan to support in bish like hash maps), but we can add `sh` code generation. It may make sense to wait on this until the language has stabilized a bit more.

**Comment by hawkw** [comment]
:+1:
